ui/apps/articles/src/pages/article-card-settings.tsx

```diff
@@ -43,8 +53,28 @@ const ArticleCardSettingsPage: React.FC<unknown> = () => {
   const { t } = useTranslation('app-articles');
   const { logger } = useRootComponentProps();
 
+  const getLicenseIcon = (type: string) => {
```

Comment:
You can use a map of type to icon element or a switch statement to make it easier to read and cleaner.